Mathlib/CategoryTheory/PUnit.lean

```diff
@@ -50,7 +50,7 @@ theorem punit_ext' (F G : C ⥤ Discrete PUnit) : F = G :=
 #align category_theory.functor.punit_ext' CategoryTheory.Functor.punit_ext'
 
 /-- The functor from `Discrete PUnit` sending everything to the given object. -/
-abbrev fromPUnit (X : C) : Discrete PUnit.{v + 1} ⥤ C :=
+abbrev fromPUnit (X : C) : Discrete PUnit.{1} ⥤ C :=
```

Comment:
How about 
```
abbrev fromUnit (X : C) : Discrete Unit ⥤ C :=
``` 
instead then? (plus some comment about why you don't want this to be polymorphic).

Reply:
I ended up making the universe of `PUnit` fully explicit everywhere in the file and then fixing the universe when defining the category of (co)structured arrows. This made more sense to me than having a single result about `Unit` when the rest of the file is about `PUnit`.